server/db/seeders/20190528204615-demo-user.js

```diff
@@ -1,13 +1,16 @@
-/* eslint-disable no-unused-vars */
```

Comment:
You need to check these files. You are overwriting the ES6 files with the old ES5 version